src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/ProbabilityTable/Builder/src/construct.hpp

```diff
@@ -0,0 +1,21 @@
+ProbabilityTable construct(){
+  try{
+    for( decltype( auto ) table : this->tables_ ){
+      details::verify::equalSize( this->incidentEnergies_.value(), table.CDFs );
```

Comment:
This is not correct. The number of incident energy values fro which tables are provided is not necessarily equal to the number of bins in teh table.

Reply:
This has been fixed.